lua/orgmode/org/autocompletion/omni.lua

```diff
@@ -9,23 +9,38 @@ local data = {
   metadata = { 'DEADLINE:', 'SCHEDULED:', 'CLOSED:' },
 }
 
-local directives = { rgx = vim.regex([[^\#+\?\w*$]]), line_rgx = vim.regex([[^\#\?+\?\w*$]]), list = data.directives }
+local directives = {
+  line_rgx = vim.regex([[^\#\?+\?\w*$]]),
+  rgx = vim.regex([[^\#+\?\w*$]]),
+  list = data.directives,
+}
+
 local begin_blocks = {
-  rgx = vim.regex([[\(^\s*\)\@<=\#+\?\w*$]]),
   line_rgx = vim.regex([[^\s*\#\?+\?\w*$]]),
+  rgx = vim.regex([[\(^\s*\)\@<=\#+\?\w*$]]),
   list = data.begin_blocks,
 }
+
 local properties = {
   line_rgx = vim.regex([[\(^\s\+\|^\s*:\?$\)]]),
   rgx = vim.regex([[\(^\|^\s\+\)\@<=:\w*$]]),
+  extra_cond = function(line, _)
+    return not string.find(line, 'file:$')
```

Comment:
Update `file:$` to `file:.*$`, since `filetags` overtakes the context after `file:` is typed.